spec/graphql/persisted_queries/schema_patch_spec.rb

```diff
@@ -39,7 +53,15 @@ def perform_request
   end
 
   context "when cache is warm" do
-    before { perform_request }
+    around do |test|
+      original = GraphqlSchema.send(:own_tracers)
```

Comment:
These tests are just a hacky placeholder for the spike.  I'd probably put these tests elsewhere in the end.